### PR TITLE
Feature/cplp-1975-remove-offer-thumbnailurl

### PR DESCRIPTION
**Pull request by qxz2mqe**
https://jira.catena-x.net/browse/CPLP-1975

### CODE REVIEW

**Review by ntruchsess — CHANGES_REQUESTED** [review]

File: src/portalbackend/PortalBackend.PortalEntities/AuditEntities/AuditOffer20221013.cs [hunk 1]
```diff
@@ -42,8 +42,6 @@ public AuditOffer20221013()
 
     public DateTimeOffset? DateReleased { get; set; }
 
-    public string? ThumbnailUrl { get; set; }
```
[the remark this answers is not in the document]
Comment (qxz2mqe) [reply src/portalbackend/PortalBackend.PortalEntities/AuditEntities/AuditOffer20221013.cs]: fixed

File: src/portalbackend/PortalBackend.Migrations/Migrations/20230119082324_CPLP-1975-RemoveOfferThumbnailUrl.cs [hunk 2]
```diff
@@ -1,23 +1,4 @@
...
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
- * License for the specific language governing permissions and limitations
- * under the License.
- *
- * SPDX-License-Identifier: Apache-2.0
- ********************************************************************************/
-
+﻿using System;
```
Comment (ntruchsess) [inline src/portalbackend/PortalBackend.Migrations/Migrations/20230119082324_CPLP-1975-RemoveOfferThumbnailUrl.cs:1]: header comment is missing

File: src/portalbackend/PortalBackend.Migrations/Migrations/PortalDbContextModelSnapshot.cs [hunk 3]
```diff
@@ -1,24 +1,4 @@
...
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
- * License for the specific language governing permissions and limitations
- * under the License.
- *
- * SPDX-License-Identifier: Apache-2.0
- ********************************************************************************/
-
-// <auto-generated />
+﻿// <auto-generated />
```
Comment (ntruchsess) [inline src/portalbackend/PortalBackend.Migrations/Migrations/PortalDbContextModelSnapshot.cs:1]: header comment is missing

File: src/portalbackend/PortalBackend.Migrations/Migrations/20230119082324_CPLP-1975-RemoveOfferThumbnailUrl.Designer.cs [hunk 4]
```diff
@@ -0,0 +1,4514 @@
+﻿// <auto-generated />
```
Comment (ntruchsess) [inline src/portalbackend/PortalBackend.Migrations/Migrations/20230119082324_CPLP-1975-RemoveOfferThumbnailUrl.Designer.cs:1]: header comment is missing

**Review by ntruchsess — CHANGES_REQUESTED** [review]

**Pull request merged**